AC3.2Unit2FinalAssessment/AC3.2Unit2FinalAssessment/DetailViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  DetailViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Annie Tung on 10/6/16.
+//  Copyright © 2016 Annie Tung. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController, UITextFieldDelegate {
```

Comment:
DetailViewController is not a proper name for this view controller. 'Detail' should be reserved for a child view controller of a tableview controller.